src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp

```diff
@@ -59,35 +59,12 @@ ParamsKey EltwiseKernel_blocked_opt::GetSupportedKey() const {
     k.EnableTensorPitches();
     k.EnableTensorOffset();
     k.EnableEltwiseBroadcast();
+    k.EnableDynamicShapesSupport();
```

Comment:
Should this kernel be selected in case of dynamic feature dimension? If so, the `LEFTOVERS` value would likely need to be handled in some special way?